Add jitpack instructions to android page (#101)
cobaltspeech/sdk-cubic: docs-src/content/using-cubic-sdk/installation.md

```

### Android

> ##### JitPack 
> 
> Add it in your root build.gradle at the end of repositories:

Reviewer: ```suggestion
#### Using Gradle and JitPack 

To use the pre-built SDK libraries, add JitPack in your root build.gradle at the end of `repositories`:
```

> ``` bash

Reviewer: ```suggestion
``` json
```
Also, fix the indentation

allprojects {
		repositories {
			...
			maven { url 'https://jitpack.io' }
		}
	}
```

> Add dependency:

Reviewer: ```suggestion
Then add sdk-cubic as a dependency:
```

> ``` bash

Reviewer: ```suggestion
``` json
```

dependencies {
    implementation 'com.github.cobaltspeech:sdk-cubic:v1.6.0'
}
```

> ##### From Souce code
> 
> Building for Android requires more steps, so it is described on the [Android Integrations](../android/) page.

Reviewer: ```suggestion
#### From Source Code

Building for Android from source requires more steps, so it is described on the [Android Integrations](../android/) page.
```

</dependencies>
```
3. You should then be able to build your project with  `mvn package`

#### Android

> #### From Source code

Reviewer 2: This was duplicated down below, so these three lines can be deleted I think.

Building for Android requires more steps, so it is described on the [Android Integrations](../android/) page.

### Android
#### Using Gradle and JitPack 
To use the pre-built SDK libraries, add JitPack in your root build.gradle at the end of `repositories`:
``` gradle
allprojects {
	repositories {
		...
		maven { url 'https://jitpack.io' }
	}
}
```
Then add sdk-cubic as a dependency:
``` gradle
dependencies {

>     implementation 'com.github.cobaltspeech:sdk-cubic:v1.6.0'

Reviewer 2: Should we add this to the version bumping script?  Or we could say something like 1+, but even that could go stale.

Author: I add replace version scripts to readme
